frontend/test/__support__/ui.tsx

```diff
@@ -191,4 +191,10 @@ export function getBrokenUpTextMatcher(textToFind: string): MatcherFunction {
   };
 }
 
+export const waitForLoadingSpinnerToDisappear = async () => {
+  await waitFor(() => {
+    expect(screen.queryByTestId("loading-spinner")).not.toBeInTheDocument();
+  });
+};
```

Comment:
Here's the util

Reply:
nit: add a comment why it's needed with the reference to a bug in RTL

Reply:
2f2e87e91423ce8b3f4365fb1db18aef62113757